Adds Italian translation

verginer opened the pull request.

verginer committed:
adds italian translation

verginer committed:
fixes quotes

verginer committed:
Update ita.json

DanielBiegler commented on ita.json:
```diff
@@ -0,0 +1,12 @@
+    "iOS8WebGLError": "WebGL ha degli errori noti su iOS 8, per questo motivo è solo possibile elaborare JPEG con encoding progressivo. (questo panorama usa un encoding standard).",
+    "genericWebGLError": "Il tuo browser non dispone del supporto necessario per WebGL per mostrare questo programma.",
+    "textureSizeError": "Questo panorama è troppo grande per il tuo dispositivo! Ha una larghezza di %spx, ma il tuo dispositivo può solo elaborare immagini con larghezza massima di %spx. Riprova con altro dispositivo. (Se sei l'autore, prova a ridimensionare l'immagine.)",
+    "unknownError": "Errore sconosciuto. Controlla la 'Developer Console'."
```
Is there no word for developer console? Google told me `console di sviluppo`. Or does that sound bad in italian?

verginer replied:
Actually that is a good translation. I personally refer to it as the "Developer Console" but looking around I have seen "console di sviluppo" being [used](https://www.chimerarevo.com/internet/cancellare-cache-singolo-sito-247144/).

DanielBiegler replied:
Cool, alright! Italian sounds so nice in my opinion, I like it very much.

verginer committed:
fixes bad wording

Developer console can be referred to as "console di sviluppo" as seen in the wild.
And corrected "mostrare" (show) to "eseguire" (execute) sounds more natural.

DanielBiegler reviewed (changes requested):
one last thing before merging, i switched to a new language code system. check the README in master for infos. So this file needs to be renamed to "IT.json" or something else if you used a specific dialect.

verginer committed:
Rename ita.json to IT.json
